timbermill-java/timbermill-local/src/main/java/com/datorama/oss/timbermill/common/disk/IndexRetryManager.java

```diff
@@ -79,14 +79,13 @@ public int retrySendDbBulkRequest(DbBulkRequest dbBulkRequest, BulkResponse resp
 			}
 		}
 		// finishing to retry - if persistence is defined then try to persist the failed requests
-		LOG.error(FLOW_ID_LOG + " Bulk #{} Reached maximum tries ({}) attempt to index.", flowId, bulkNum, numOfElasticSearchActionsTries);
+		LOG.warn(FLOW_ID_LOG + " Bulk #{} Reached maximum tries ({}) attempt to index.{}", flowId, bulkNum, numOfElasticSearchActionsTries, hasPersistence()? " Bulk will be persist to disk":"");
 		tryPersistBulkRequest(dbBulkRequest, flowId, bulkNum);
```

Comment:
it's not related to the task, 
I changed a little bit the logs of the db

Reply:
Let's keep it an ERROR, I want to get an alert when we persist to disk.